DependencyInjection/SonataAdminExtension.php

```diff
@@ -85,8 +85,23 @@ public function load(array $configs, ContainerBuilder $container)
         $processor = new Processor();
         $config = $processor->processConfiguration($configuration, $configs);
 
-        $config['options']['javascripts'] = $config['assets']['javascripts'];
-        $config['options']['stylesheets'] = $config['assets']['stylesheets'];
+        $javascripts = array_unique(
+            array_merge(
+                $config['assets']['javascripts'],
+                $config['assets']['extra_javascripts']
```

Comment:
i tested it and and arrayNode() which is not required should be handeled by the configuration itself IMO

but i'll check that again